### PR TITLE
Access Filters + Tag by Variant Classification pv commands

### PR DESCRIPTION
**Pull request by rnaidu**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/msk/nucleovar/tree/master/.github/CONTRIBUTING.md)
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).
- [x]  recalculate the extract_tn_genotype function statistics for the matched normal sample instead of the control sample
- [x] create grouping for curated and plasma groups (duplex, and simplex-duplex). change calculations for them as necessary
- [x] create a [helper.py](http://helper.py/) just for filter functions and move all access_filters functions to there
- [x] add documentation for each function in access_filters
- [x] updated df_condensed so it includes all cols up until n_vaf_fragment
- [x] run comparison test on MAF files generated with full test set from voyager run
- [x] make sure normal category sample statistics match
- [x] create tag_by_variant_classification command in maf tag 
- [x] add preprocessing portion of the maf2tsv script as a function in access filters 
- [x] create separate mini tagging functions for each text file created in tag by variant classification  (exonic,silent,nonpanel_exonic,nonpanel_silent,dropped). see lucidchart for exact logic structure 
- [x] turn exonic text file into a MAF output too (so it can be used for the IGV loader script)

### CODE REVIEW

File: tests/test_maf.py [hunk 1]
```diff
@@ -69,6 +69,7 @@
     ["maf", "tag", "prevalence_in_cosmicDB", "--help"],
     ["maf", "tag", "truncating_mut_in_TSG", "--help"],
     ["maf", "tag", "traceback", "--help"],
+    ["maf", "tag", "by_variant_classification", "--help"],
 ]
```
Comment (buehlere) [inline tests/test_maf.py:73]: add `--help` test for `access_remove_variants`

File: postprocessing_variant_calls/maf/annotate/workflow.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+changes being made to annotation process functions
```
Comment (buehlere) [inline postprocessing_variant_calls/maf/annotate/workflow.txt:1]: remove

File: out_file.md [hunk 3]
```diff
@@ -0,0 +1,628 @@
+# `main`
+
```
Comment (buehlere) [inline out_file.md:2]: remove

**Review by buehlere — CHANGES_REQUESTED** [review]
just a few small things to cleanup, but then should be ready to go!

**Comment by rnaidu** [comment]
> just a few small things to cleanup, but then should be ready to go!

Thanks for reviewing! Changes have been made and PR is ready to be merged into develop

**Review by buehlere — APPROVED** [review]
Looks good!

**Pull request merged**